ripper/services.py

```diff
@@ -142,14 +162,22 @@ def validate_input(args) -> bool:
         print(f'Wrong attack type. Possible options: udp, tcp, http.')
         return False
 
+    if args.http_method and args.http_method.lower() not in ('get', 'post', 'head', 'put', 'delete', 'trace', 'connect', 'options', 'patch'):
+        print(f'Wrong http method type. Possible options: get, post, head, put, delete, trace, connect, options, patch.')
+        return False
+
+    if args.http_path and not args.http_path.startswith('/'):
```

Comment:
For Windows users, we must to check path delimiter with OS specifics in mind. This check should be reworked to support OS specific path delimiter.